### PR TITLE
Adding Event Feed to the Event Service

### PR DESCRIPTION
**Pull request by lancewf**
### :nut_and_bolt: Description

Adding the Event Feed service to the Event Service. This pushes compliance events to the new event feed as well as the old one. 

### :athletic_shoe: Demo Script / Repro Steps

1. `build components/event-service && start_all_services`
1. Add an event. Go to https://a2-dev.test/compliance/compliance-profiles and import a profile. 
1. Test that the event was added to the event-feed-service with `chef-automate dev grpcurl event-feed-service -- chef.automate.domain.event_feed.api.EventFeedService.GetFeed -d '{"Size": 100}'` or `curl $ELASTICSEARCH_URL/eventfeed-0-feeds/_search | jq . | less`

### :chains: Related Resources

https://github.com/chef/automate/issues/444

### :white_check_mark: Checklist

- [ ] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [x] Code actually executed?
- [ ] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

File: components/event-service/server/server.go [hunk 1]
```diff
@@ -55,12 +55,12 @@ func New(cfg *config.EventConfig) *Server {
...
-	registry[ScanJobDeleted] = []string{config.FEED_KEY}
-	registry[ProfileCreated] = []string{config.FEED_KEY}
-	registry[ProfileUpdated] = []string{config.FEED_KEY}
-	registry[ProfileDeleted] = []string{config.FEED_KEY}
+	registry[ScanJobCreated] = []string{config.FEED_KEY, config.EVENT_FEED}
+	registry[ScanJobUpdated] = []string{config.FEED_KEY, config.EVENT_FEED}
+	registry[ScanJobDeleted] = []string{config.FEED_KEY, config.EVENT_FEED}
+	registry[ProfileCreated] = []string{config.FEED_KEY, config.EVENT_FEED}
+	registry[ProfileUpdated] = []string{config.FEED_KEY, config.EVENT_FEED}
+	registry[ProfileDeleted] = []string{config.FEED_KEY, config.EVENT_FEED}
```
Comment (lancewf) [inline components/event-service/server/server.go]: For now, we are going to send these events to both the old event feed and the new one.

File: components/event-service/event/events.go [hunk 2]
```diff
@@ -177,6 +178,18 @@ func (svc Events) getClient(handlerType string) (EventHandlerClient, error) {
 			return nil, errors.New("CallHandler could not obtain NewProjectsClient")
 		}
 		return authzProjectsClient, nil
+	} else if handlerType == config.EVENT_FEED {
```
Comment (srenatus) [inline components/event-service/event/events.go]: [nit] I know you're just adding one, but wouldn't this make sense as a `switch` statement?

Comment (lancewf) [reply components/event-service/event/events.go]: I don't see why not to use a `switch` I will try it out.

File: components/event-service/config/config.go [hunk 3]
```diff
@@ -139,6 +142,7 @@ func Configure() (*EventConfig, error) {
 	log.Debugf("feed endpoint: %s", config.HandlerEndpoints.Feed)
 	log.Debugf("cfg ingest endpoint: %s", config.HandlerEndpoints.CfgIngest)
 	log.Debugf("authz endpoint: %s", config.HandlerEndpoints.Authz)
+	log.Debugf("event feed endpoint: %s", config.HandlerEndpoints.EventFeed)
```
Comment (phiggins) [inline components/event-service/config/config.go]: Why log this twice in the same function?

Comment (lancewf) [reply components/event-service/config/config.go]: I am not sure why :man_shrugging: I was just following along with the other endpoints. I am going to remove this bottom set of logs for the endpoints.

**Pull request merged**